src/app/component/eco-news/create-news/drag-and-drop/drag-and-drop.component.html

```diff
@@ -8,7 +8,27 @@
 </div>
 <div *ngFor="let file of files">
   <img 
-	*ngIf="file"
+	*ngIf="file && 
+	file.file.size < 10485760 && 
+	(file.file.type === 'image/jpeg' || file.file.type === 'image/png'); 
+	else elseBlock" 
 	[src]="file.url" 
```

Comment:
it`s very complicated to understand file.file please change it to proper names

Reply:
fixed